lib/packer/instance.rb

```diff
@@ -38,5 +38,17 @@ def dev_server
     def commands
       @commands ||= Packer::Commands.new self
     end
+
+    private
+
+    def app_root_path
+      if Packer.rails?
+        Rails.root
+      elsif Packer.sinatra?
+        Sinatra::Base.settings.root || Pathname('.')
+      else
+        raise 'Packer is running in unsupported environment'
```

Comment:
What exception type does this raise? `StandardError`?

Reply:
This would raise `RuntimeError`.

Reply:
Ahh ok 👍